content/en-us/designs/protocol-translation/_index.md

```diff
@@ -0,0 +1,198 @@
+
+## Example - Industrial IoT Data Extraction
+This is an example of implementing a protocol translation on a gateway in the context of Industrial IoT data extraction.
+
+> Note these examples are not production ready code. They exist for the purpose of education.
+
+Industrial machines are often equipped with, or consist of, sensors and other hardware such as programmable logic controllers ([PLC]({{< ref "/glossary/vocabulary#plc" >}})) that are used to monitor or control machine behavior.
+As a prerequisite for a large set of [Industry 4.0]({{< ref "/glossary/vocabulary#industry40" >}}) use cases, it is required to access the data of industrial machines and their components.
+The two main main reasons why simply connecting these machines to a network or the internet is not viable are in line with the challenges mentioned above:
+* **Protocols:** There are many different communication standards for the aforementioned components of industrial machines, e.g., [OPC-UA](https://en.wikipedia.org/wiki/OPC_Unified_Architecture), [Modbus](https://en.wikipedia.org/wiki/Modbus), [CANBus](https://en.wikipedia.org/wiki/CAN_bus), [Profibus](https://en.wikipedia.org/wiki/Profibus).
+Some of these standards have not been designed for use over the internet and some, e.g., serial Modbus, are not even based on the [IP-Protocol](https://en.wikipedia.org/wiki/Internet_Protocol).
+* **Security:** Connecting an industrial asset directly to the internet is a security risk. For instance, Modbus in its basic version does not have any kind of security mechanism to authenticate a system that reads or writes to the underlying industrial equipment.
+
+To overcome these challenges the application of the protocol translation design requires the deployment of a [Gateway]({{< ref "/designs/gateway" >}}) to connect the machines as well as to provide a secure connection to the cloud.
+The local network on the left-hand of the following diagram is commonly called the *shop floor* in the manufacturing, or *operational technology (OT)* domains.  
+
+![IIoT Extraction Design](IIoT-Extraction-Architecture.png)
+
+The gateway itself contains translation logic that translates between the machine protocol and the common protocol used by IT systems.
+In the example above, there are two machines - **`A`** and **`B`**.
+Suppose `Machine A` has an interface that follows the [OPC-UA standard](https://en.wikipedia.org/wiki/OPC_Unified_Architecture) and `Machine B` uses [Modbus](https://en.wikipedia.org/wiki/Modbus).
+The following pseudo-code samples demonstrate the protocol translation logic that needs to run on the gateway.
+
+#### Protocol Translation OPC-UA
+OPC-UA is a modern, IP-based protocol that has security features like in-transit encryption and authentication using certificates or username/password combinations.
+This pseudo-code shows how to access an OPC-UA target in Python syntax: 
+
+```python2
```

Comment:
For code block, `python2` not valid for syntax code highlighting, please replace with just `python`